packages/wallet/src/redux/protocols/application/actions.ts

```diff
@@ -17,8 +17,8 @@ export interface OpponentCommitmentReceived {
   commitment: Commitment;
   signature: string;
 }
-export interface ConcludeRequested {
-  type: 'WALLET.APPLICATION.CONCLUDE_REQUESTED';
+export interface Concluded {
```

Comment:
I'm not sure I understand the name change here. `ConcludeRequested` seems more descriptive of what's actually happening, unless I've misunderstood?

Reply:
'WALLET.NEW_PROCESS.CONCLUDE_REQUESTED' still exists, but the point is that it spins up a protocol which can *fail* (by the user backing out / cancelling). The `requested` part seems apt for this action.

From the application reducer's point of view, we don't want to do anything when the user requests a conclude; we only want to terminate and stop signing commitments once we know for sure that the channel is `Concluded`.